src/olympia/stats/views.py

```diff
@@ -170,7 +165,10 @@ def iterator(series):
             item = next(series)
             next_date = start_date
             while True:
-                if item['date'] == next_date:
+                date = item['date']
+                if isinstance(date, str):
+                    date = datetime.strptime(date, '%Y-%m-%d').date()
```

Comment:
same here.  We're now converting string to date... but only when it's a str (what other types can it be?)